apps/bfd-data-npi/README.md

```diff
@@ -0,0 +1,25 @@
+BFD NPI to Organization Name Lookup
+=======================================
+
+This maven project produces a jar file that provides functionality to retrieve organization names given an NPI as input. This lookup is backed by a data file stored in the jar files as a resource.
+
+The data file is constructed from 
```

Comment:
suggest: `The data file is constructed from a pipe delimited file an example of which is given below`

Also, lets provide a link to that NPI landing page and the most recent CSV file for reference

Reply:
done